app/assets/stylesheets/story_show_page.scss

```diff
@@ -1,3 +1,7 @@
 .personal-image {
   margin-bottom: 2rem;
 }
+
+.simple_form input[type='submit'] {
+  color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.